Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -0,0 +1,380 @@
+   the configuration for the current plugin.
+
+Steps to Perform (Extension Developer)
+======================================
+
+.. rst-class:: bignums-xxl
+
+#. Create configuration schema
+
+   A configuration schema in XML is added to the extension.
+
+   Example: :file:`Configuration/FlexForms/Registration.xml`.
+
+   .. code-block:: xml
+
+        <?xml version="1.0" encoding="utf-8" standalone="yes" ?>
+        <T3DataStructure>
+            <sheets>
+                <sDEF>
+                    <ROOT>
+                        <TCEforms>
+                            <sheetTitle>LLL:EXT:extkey/Resources/Private/Language/Backend.xlf:settings.registration.title</sheetTitle>
+                        </TCEforms>
+                        <type>array</type>
+                        <el>
+                            <!-- Add settings here ... -->
+
+                            <!-- Example setting: checkbox with name settings.includeCategories -->
+                            <settings.includeCategories>
+                                <TCEforms>
+                                    <label>LLL:EXT:example/Resources/Private/Language/Backend.xlf:settings.registration.includeCategories</label>
+                                    <config>
+                                        <type>check</type>
+                                    </config>
+                                </TCEforms>
+                            </settings.includeCategories>
+
+                            <!-- end of settings -->
+
+                        </el>
+                    </ROOT>
+                </sDEF>
+            </sheets>
+        </T3DataStructure>
+
+#. Add texts and translations
+
+   Add text for the language keys used in schema (LLL) to your language
+   files in :file:`Resources/Private/Languages`.
+
+   .. code-block:: xml
+
+       <trans-unit id="settings.registration.title">
+           <source>Settings for Registration plugin</source>
+       </trans-unit>
+       <trans-unit id="settings.registration.includeCategories">
+           <source>Include categories</source>
+       </trans-unit>
+
+
+#. The configuration schema is attached to one or more plugins
+
+   Here, the vendor name is **Myvendor, the extension key is **example**
+   and the plugin name is **Registration**.
+
+   in :file:`Configuration/TCA/Overrides/tt_content.php`:
+
+   .. code-block:: php
+
+      \TYPO3\CMS\Extbase\Utility\ExtensionUtility::registerPlugin(
+          // The extension name (in UpperCamelCase) or the extension key (in lower_underscore)
+          'Myvendor.example',
+          // Plugin name: unique id in UpperCamelCase
+          'Registration',
+          // title
+          'LLL:EXT:example/Resources/Private/Language/Backend.xlf:registration_title'
+      );
+
+   * `API reference: registerPlugin <https://github.com/TYPO3/TYPO3.CMS/blob/ddedb065030de860de87e39636ed2664c7c531cd/typo3/sysext/extbase/Classes/Utility/ExtensionUtility.php#L104>`__
```

Comment:
I would add the urls above the example. At least for me it's not easy to align the urls to the example above.

Maybe something like "use `registerPlugin` in :file:`...

Also note that we can add deeplinks to api.typo3.org through intersphinx. At least I would not reference a concrete commit. Better reference matching TYPO3 version. Even this has to be adjusted within documentation after each release, but could be easily grepped and adjusted. Still adds further work and is another task that we probably will forget.